### ISSUE TITLE
Publish gem to Rubygems

### ISSUE THREAD

**Issue by jonspalmer** [arcadia/omniauth-oauth2-cognito#2]
It would be terrific to publish this gem to Rubgmes so that that git url is not required as the source in the Gemfile.

This is important for folks in environment's where Rubygems, npm etc. are proxied or mirrored and direct access to github is more difficult.

**Comment by vozeldr** [comment]
Github has (since like November) the ability to act as a repository for packages, including rubygems (also npm, maven, docker, nuget). Not sure if that info helps.

**Comment by jonspalmer** [comment]
@vozeldr Yes it sort of helps but the challenge is this Gem should really be setup so that we can add is simply with:
```ruby
gem 'omniauth-oauth2-cognito'
```

**Comment by rrjohnson85** [comment]
@jonspalmer this can be done pretty easily without publishing to rubygems or Github.

Add this to you gemfile:
```
git_source(:github) { |repo| "https://github.com/#{repo}.git" }
```

The library can then be added by:
```
gem 'omniauth-oauth2-cognito', github: 'arcadia/omniauth-oauth2-cognito'
```

**Comment by rrjohnson85** [comment]
Some more details can be found here, https://bundler.io/guides/git.html.
